micromedio/herz-ui
# feat(Checkbox): grow label to container size

### What kind of PR is this?

### What is this PR for?
The label element doesn't grow to the size of the checkbox container

### How was this achieved?
Added flex-grow: 1 to the component label

### What areas of the application does it impact?
Checkbox element

### How to test?
Should be able to click on the container where there is no text and check the box
